src/modules/drawing/components/LineStylePopup.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div class="line-style-popup">
+        <button type="button" class="close" @click="onClose">
+            <span aria-hidden="true">&times;</span>
+        </button>
+
+        <color :color="color" :colors="options.colors" @onColorChange="onColorChange"></color>
```

Comment:
Could you use the uppercase name here? `<Color ...></Color>`
This way it's easier to know this is one of our custom component and not some custom tag coming from another library